images/generic/flatcar.yaml

```diff
@@ -1,21 +1,3 @@
 download_images: true
 build_name: "flatcar-stable"
 packer_builder_type: ""
-
-containerd_version: 1.5.4 # match this with the flatcar distribution version
-containerd_sha256: 591e4e087ea2f5007e6c64deb382df58d419b7b6922eab45a1923d843d57615f # matching sha to the version override above
-kubernetes_cni_version: 0.8.7
```

Comment:
Was removing these intentional?
Do we want to keep it at `1.5.4` for Flatcar until we can catch up in our package manager?

Reply:
I think 1.5 might be required to allow docker CLI to function with containerd. Don't quote me on that, but it's been working with 1.5.4 since we released it. Given that, we do need to update containerd soon across the board, so maybe flatcar is a good test bed for that.

Reply:
the containerd flatcar comes with is 1.5.4

```
core@ip-10-0-164-70 ~ $ containerd --help
NAME:
   containerd - 
                    __        _                     __
  _________  ____  / /_____ _(_)___  ___  _________/ /
 / ___/ __ \/ __ \/ __/ __ `/ / __ \/ _ \/ ___/ __  /
/ /__/ /_/ / / / / /_/ /_/ / / / / /  __/ /  / /_/ /
\___/\____/_/ /_/\__/\__,_/_/_/ /_/\___/_/   \__,_/

high performance container runtime


USAGE:
   containerd [global options] command [command options] [arguments...]

VERSION:
   1.5.4

```